incubator/jenkins-operator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 0.1.12
+appVersion: 0.1.15
 description: A Helm chart for Kubernetes Jenkins Operator
 name: jenkins-operator
-version: 0.1.12
+version: 0.1.15
```

Comment:
This is a breaking change for the chart. To follow SemVer this chart needs to be `0.2.0` for the version. Can you please increment.